recipes/datasets/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "datasets" %}
+{% set version = "1.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 40261f45806ebe003194bb6d14b3f59a6f1e7f9e347e78b662e1ab979ace7e9c
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3,<3.7
+  run:
+    - python >=3,<3.7
+    - dill
+    - multiprocess
+    - numpy >=1.17
+    - pandas
+    - pyarrow >=0.17.1
+    - python
+    - requests >=2.19.0
+    - tqdm >=4.27,<4.50.0
+    - python-xxhash
+    - dataclasses
+
+test:
+  imports:
+    - datasets
+    - datasets.commands
+    - datasets.utils
+
+about:
+  home: https://github.com/huggingface/datasets
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE
```

Comment:
I opened a PR upstream to fix having to manually package the license file (https://github.com/huggingface/datasets/pull/1007). Please add the following comment as a visual reminder to check the status of whether the license file is in the source distribution:

```yaml
# License file manually packaged. See: https://github.com/huggingface/datasets/pull/1007
license_file: LICENSE
...
```